compiler/src/codegen.rs

```diff
@@ -564,10 +597,16 @@ impl<'a> ServiceGen<'a> {
         ServiceGen { proto, methods }
     }
 
+    #[cfg(feature = "protobuf-codec")]
     fn service_name(&self) -> String {
         util::to_camel_case(self.proto.get_name())
     }
 
+    #[cfg(feature = "protobufv3-codec")]
+    fn service_name(&self) -> String {
+        util::to_camel_case(self.proto.name)
```

Comment:
You could move the conditional into the method here and avoid duplicating the method signature.

Reply:
Done, thanks